### DISCUSSION TITLE
upload large file causing Error: "Payload content length greater than maximum allowed: 1048576"

### DISCUSSION THREAD

**Discussion by shivika24** [serverless/serverless#10547] [Q&A]
I am just trying to invoke one lambda function from another lambda function, and i am sending a payload into that lambda function. If I paas a payload of size upto 1 mb, then i don't get this error, this error arises when the payload size is greater than 1 mb only.

`import * as AWS from "aws-sdk";

const lambda = new AWS.Lambda({
  region: "us-east-1",
  endpoint: "http://localhost:3002",
});

const params = {
  FunctionName: "myFunction", // the lambda function we are going to invoke
  InvocationType: "RequestResponse",
  LogType: "Tail",
  Payload: JSON.stringify(data),
 };

const lambdaResult = await lambda.invoke(params).promise();
`

**Comment by pgrzesik** [comment]
Hello @shivika24 - as the error message suggests, the payload is too big and invoking function this way with such payload is just not supported.

**Comment by shivika24** [comment]
@pgrzesik, but according to aws documentation, it is mentioned that payload of size upto 6mb for a lambda function is allowed.

**Reply by pgrzesik** [reply]
It looks like you're using some kind of local emulation - please consult the docs of that emulation platform then.

**Reply by shivika24** [reply]
I am just using serverless locally, to invoke lambda functions, nothing else.

**Reply by pgrzesik** [reply]
I see that you're specifying `endpoint: "http://localhost:3002",` option - what is the reason for that?

**Reply by shivika24** [reply]
To invoke the lambda's locally i am specifying the endpoint, as i have not deployed my lambda's yet.

**Reply by pgrzesik** [reply]
But what are you using as a way to emulate Lambda service locally? Localstack, something similar?
